changelog.txt

```diff
@@ -1,5 +1,17 @@
 *** Changelog ***
 
+2020.01.14 - version 2.3.0
+* New: Configure Renovate - #2805
```

Comment:
Do we usually put non-product changes in Sensei's changelog?

Reply:
Hrm...I wasn't before but I saw some plugins do. What do you think?

Reply:
If I was a user, I might be a little confused. With `New: ` prefix, especially, I might wonder how I can now configure Renovate in Sensei (and what that means 😉). As a user myself I tend to look at changelogs in terms of new features, product changes that might impact my usage, or bug fixes, not necessarily behind-the-scenes changes.

Reply:
OK, since I merged before seeing your comment, I'll make a note to remove related items from the change log when I create the release PR.